docs/source/tutorial/visualise_pipeline.md

```diff
@@ -233,3 +235,68 @@ You can view the larger visualisation of the chart by clicking the 'Expand Plotl
 
 
 ![](../meta/images/pipeline_visualisation_plotly_expand.png)
+
+
+### Visualise Matplotlib charts in Kedro-Viz
+
+[Matplotlib](https://matplotlib.org/) is a Python library for creating static, animated, and interactive visualisations. Integrating Matplotlib into Kedro-Viz allows you to output your charts as part of your pipeline visualisation.
+
+```{note}
+The MatplotlibWriter dataset converts Matplotlib objects to image files. This means that Matplotlib charts within Kedro-Viz are static and not interactive, unlike the Plotly charts seen above.
+```
+
+You can view Matplotlib charts in Kedro-Viz when you use the [Kedro MatplotLibWriter dataset](/kedro.extras.datasets.matplotlib.MatplotlibWriter). You must update the `src/requirements.txt` file in your Kedro project by adding the following dataset to enable Matplotlib for your project:
+
+```
+kedro[matplotlib.MatplotlibWriter]==0.18.2
+```
+
+To use this dataset, configure your plot in your Kedro node. The below functions can be added to the `nodes.py` and `pipeline.py` files respectively.
+
+```python
+# nodes.py
+import matplotlib.pyplot as plt
+
+
+def create_confusion_matrix(companies: pd.DataFrame):
+    actuals = [0, 1, 0, 0, 1, 1, 1, 0, 1, 0, 1]
+    predicted = [1, 1, 0, 1, 0, 1, 0, 0, 0, 1, 1]
+    data = {"y_Actual": actuals, "y_Predicted": predicted}
+    df = pd.DataFrame(data, columns=["y_Actual", "y_Predicted"])
+    confusion_matrix = pd.crosstab(
+        df["y_Actual"], df["y_Predicted"], rownames=["Actual"], colnames=["Predicted"]
+    )
+    sn.heatmap(confusion_matrix, annot=True)
+    return plt
+
+
+# pipeline.py
+def create_pipeline(**kwargs) -> Pipeline:
+    """This is a simple pipeline which generates a plot"""
+    return pipeline(
+        [
+            node(
+                func=create_confusion_matrix,
+                inputs="companies",
+                outputs="dummy_confusion_matrix",
+            ),
+        ]
+    )
+```
+
+You must also specify the output type in the `catalog.yml` file, like below.
+
+```yaml
+reporting.dummy_confusion_matrix:
+  type: matplotlib.MatplotlibWriter
+  filepath: ${base_location}/08_reporting/dummy_confusion_matrix.png
+  versioned: true
```

Comment:
It might be useful to add a sentence about why `versioned: true` has been set here.

Reply:
Good point! Added an explanation in [14d7b98](https://github.com/kedro-org/kedro/pull/1820/commits/14d7b98411fb5e60c849fc7e5e278a3bf370ec38).